.prettierrc.json

```diff
@@ -2,5 +2,6 @@
   "$schema": "https://json.schemastore.org/prettierrc.json",
   "singleQuote": true,
   "printWidth": 100,
-  "plugins": ["@prairielearn/prettier-plugin-sql"]
+  "plugins": ["prettier-plugin-sh", "@prairielearn/prettier-plugin-sql"],
+  "indent": 4
```

Comment:
I set this to `4` -- but it probably warrants further discussion in #dev-core. Most of our scripts currently use 4.

Reply:
I think this is fine.

Prettier claims it will grab this from `.editorconfig`. Can we remove this from `.prettierrc.json`?

Reply:
If I remove this, then `sh` snippets on the docs go to 2 spaces.

Reply:
I think that's fine?

Reply:
Agreed 👍